src/controllers.js

```diff
@@ -0,0 +1,35 @@
+import crypto from 'crypto';
+import { isEmpty, isString } from 'lodash';
+
+import { getConnection } from './database';
+
+export const pingController = (req, res, next) => res.json('pong');
+
+export const authorizeController = async (req, res, next) => {
+  try {
+    const apiKey = req.get('X-API-Key');
+
+    if (!isString(apiKey)) {
+      throw new Error('api key not found');
+    }
+
+    const databaseConnection = await getConnection();
```

Comment:
`databaseConnection` doesn't seem explicit enought. What do you think about : 'ExistingTokenList' ?